guides/source/configuring.md

```diff
@@ -277,7 +277,7 @@ Every Rails application comes with a standard set of middleware which it uses in
 * `Rack::Runtime` sets an `X-Runtime` header, containing the time (in seconds) taken to execute the request.
 * `Rails::Rack::Logger` notifies the logs that the request has begun. After request is complete, flushes all the logs.
 * `ActionDispatch::ShowExceptions` rescues any exception returned by the application and renders nice exception pages if the request is local or if `config.consider_all_requests_local` is set to `true`. If `config.action_dispatch.show_exceptions` is set to `false`, exceptions will be raised regardless.
-* `ActionDispatch::RequestId` makes a unique X-Request-Id header available to the response and enables the `ActionDispatch::Request#uuid` method.
+* `ActionDispatch::RequestId` makes a unique X-Request-Id header available to the response and enables the `ActionDispatch::Request#uuid` method. The header name can be changed from `X-Request-Id` by setting `config.action_dispatch.request_id_header` to the intended value.
```

Comment:
"Configurable with `config.action_dispatch.request_id_header`." is sufficient and matches the other middleware descriptions.

Reply:
Cheers, updated to match